packages/server/graphql/private/mutations/updateEmail.ts

```diff
@@ -10,19 +10,39 @@ const updateEmail: MutationResolvers['updateEmail'] = async (_source, {oldEmail,
     throw new Error('New email is the same as the old one')
   }
 
-  const user = await getUserByEmail(oldEmail)
+  const user = await pg
+    .selectFrom('User')
+    .selectAll()
+    .where('email', '=', oldEmail)
+    .executeTakeFirst()
+
   if (!user) {
     throw new Error(`User with ${oldEmail} not found`)
   }
 
-  // RESOLUTION
+  //Use identities array for the user above
+  const identitiesArray = user.identities
+
+  // Update the identities array for AuthIdentityLocal, isEmailVerified=false
+  if (identitiesArray && identitiesArray.length > 0) {
```

Comment:
TBH this mutation should be rewritten if we start dipping into identities. There's an assumption that the identity stays the same, only the email changes, which we can only verify by talking to the user. This mutation should work for SSO or google identities, too, which this new code will break.

I'd suggest we hold off for now as this work isn't eating our lunch, but breaking current functionality could impact our customer service folks. In a better future we'll just enforce email verification upon signup so we can reduce the number of spam account & do away with the flag all together

Reply:
Alternatively, we could also remove just the else part of the code, throwing no error.
This would not break the existing functionality and keep working for LOCAL identities. 
As and when SSO identities are implemented, this mutation can also be updated.

Reply:
totally down for that! as long as API doesn't break current functionality I'm happy

Reply:
Great! I have update the PR accordingly, tested the same; works for both Google Auth and Local.